MyCryptoHQ/MyCrypto
# Fix/ts ctx menu

### Description

Fixes CI Error present in #1955 that was uncaught before

### Changes

* Pass in correct parameters for `.popup` method

### Steps to Test

1. Test the right-click context menu implemented in https://github.com/MyCryptoHQ/MyCrypto/pull/1780
2. Make sure the right click menu works as shown in the screenshot of  #1780
